src/StateSmith/Output/Gil/C99/C99GenVisitor.cs

```diff
@@ -530,7 +548,7 @@ public override void VisitIdentifierName(IdentifierNameSyntax node)
 
         switch (result)
         {
-            case "Boolean": result = "bool"; break;
+            case "Boolean": result = renderConfigC.UseStdBool ? "bool": "int"; break;
```

Comment:
Is there any advantage to use uint8_t instead of target integer type?

Reply:
the only thing I can think of is that it may use a bit less stack in debug builds. This could be helpful in small systems. We could also use `char` or `unsigned char`.

Reply:
Or maybe changing the boolean value by a string allowing to provide the wanted type?

Reply:
interesting idea. Is there a reason you would prefer `int` over a small type like `char`?

Reply:
We can leave as int for now. No need to stall the pr. We can revisit if there comes a need in the future.